.github/workflows/rust.yml

```diff
@@ -193,8 +211,8 @@ jobs:
   audit:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - uses: EmbarkStudios/cargo-deny-action@v2
+      - uses: actions/checkout@v4
+      - uses: EmbarkStudios/cargo-deny-action@v2
```

Comment:
I suggest doing these formatting changes in a separate pull request.

Reply:
I believe it's done accidentally since I have "Format On Save" enabled.